editor/plugins/abstract_polygon_2d_editor.cpp

```diff
@@ -778,24 +797,23 @@ AbstractPolygon2DEditor::AbstractPolygon2DEditor(EditorNode *p_editor, bool p_wi
 	selected_point = Vertex();
 	edge_point = PosVertex();
 
+	disable_polygon_editing(false, String());
```

Comment:
This call is too early, the buttons don't exist yet.

Reply:
I'd put this in `NOTIFICATION_READY` instead.

Reply:
Done, see #25958.